pkg/pillar/containerd/oci.go

```diff
@@ -217,3 +221,29 @@ func (s *ociSpec) updateFromImageConfig(config v1.ImageConfig) error {
 	}
 	return oci.WithAdditionalGIDs("root")(ctrdCtx, CtrdClient, &dummy, &s.Spec)
 }
+
+// UpdateMounts
+func (s *ociSpec) UpdateMounts(disks []types.DiskConfig) {
+	for i, disk := range disks {
+		// Skipping root container disk
+		if i == 0 {
+			continue
+		}
+		mount := specs.Mount{}
+		access := ""
+		if disk.ReadOnly {
+			access = "rbind:ro"
+		} else {
+			access = "rbind:rw"
+		}
+		mount.Type = "bind"
+		mount.Source = path.Join("/var", disk.FileLocation)
+		if disk.MountDir != "" {
+			mount.Destination = disk.MountDir
+		} else {
+			mount.Destination = "/host"
```

Comment:
I am confused about the case where we have a few Volumes without explicit disk.MountDir set. Should't we have some kind of a scheme where all of them get mounted to something more like /mnt/VOLUME NAME?

Reply:
Resolved. If MountDir is empty for volume then we will mount it on /mnt/<volume-name>